### PR TITLE
Hypo fix

### PR DESCRIPTION
**Pull request by TGCMbetcher**
## About The Pull Request

A quick typo fix that Im readding since I messed up some merges and couldnt fix the last one.

## Why It's Good For The Game

It makes some sentences make sense and adds uniformity to item names.

## Changelog
:cl:
fix: fixed typos and names
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/xenomorph.dm [hunk 1]
```diff
@@ -225,7 +225,7 @@
 	if(ishuman(L))
 		if(!chestburst)
 			do_attack_animation(L, ATTACK_EFFECT_GRAB)
-			if(L.stat == DEAD && !L.headbitten) //Grab delay vs the non-headbitten dead
+			if(!L.headbitten) //Grab delay vs the non-headbitten dead
 				if(!do_mob(src, L , XENO_PULL_CHARGE_TIME, BUSY_ICON_HOSTILE))
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/xenomorph.dm:229]: uhhhhhhhhhhh

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by TGCMbetcher** [comment]
How did this happen again, this is its own separate push. I pushed it out separately, I had it have its own branch, the other was already pushed, they arent merged.

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -279,38 +279,38 @@
...
+	name = "Meralyne hypospray"
+	desc = "A hypospray loaded with meralyne."
 	list_reagents = list(
 		/datum/reagent/medicine/meralyne = 60,
 	)
 
 /obj/item/reagent_containers/hypospray/advanced/dermaline
-	name = "A dermaline hypospray"
-	desc = "An hypospray loaded with dermaline."
+	name = "Mermaline hypospray"
```
Comment (BraveMole) [inline code/game/objects/items/reagent_containers/hypospray.dm:305]: You mean Dermaline ?

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by TGCMbetcher** [comment]
Right, so read over them more before I push put and you can edit here to very easily change things. Nice to know. Even learned how to delete edits for once.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**